tests/src/cli-test.js

```diff
@@ -72,21 +73,21 @@ test('cli :: returns exit code without menu when --no-interactive is set', async
   t.equal(await cli.execute([nodeBin, nibbleBin, okayPath, '--no-interactive']), 0, 'exits with 0 if no problems');
   console.log = function (input) {
     console.log = origConsoleLog;
-    t.ok(input.includes('No lint failures found for rule(s): prefer-const', 'Warns user'));
+    t.ok(stripAnsi(input).includes('No lint failures found for rule(s): prefer-const', 'Warns user'));
```

Comment:
Any idea why this started to fail after the changes here?  I'm a bit surprised that they would have passed before and started failing now, since I think you just copied over the code from the dependency.  Maybe it's now using an updated version of chalk or text-table, which is changing the output?

Reply:
Not sure, I checked out master, and they appear to have been failing on master as well. I'll double check myself on that however. 👍🏻

Reply:
Are you on Windows or something, by chance?  I checked out master and it passed fine, and I removed these from your branch and it works fine too.  But maybe your environment is different.